Rubberduck.VBEditor.VBA/SourceCodeHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.IO;
+using System.Text;
+using Rubberduck.Resources;
+using Rubberduck.VBEditor.SafeComWrappers;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+
+namespace Rubberduck.VBEditor.VBA
+{
+    public class SourceCodeHandler : ISourceCodeHandler
+    {
+        public string Export(IVBComponent component)
+        {
+            var fileName = component.ExportAsSourceFile(ApplicationConstants.RUBBERDUCK_TEMP_PATH);
+
+            return File.Exists(fileName) 
+                ? fileName 
+                : null; // a document component without any code wouldn't be exported (file would be empty anyway).            
+        }
+
+        public void Import(IVBComponent component, string fileName)
+        {
+            var components = component.Collection;
```

Comment:
This code is unchanged from previously (simply moved) - but does components need to be in a using block here?

Reply:
IMO, yes, the `components` variable should be in a `using` block.